File: frontend/routes/messenger.route.js
const router = require("express").Router();

const {
  middlewareCheckGoogleProfile,
  middlewareCheckEmail,
} = require('../controllers/oauth-google.controller');

const {
  getMemberChat,
} = require('../controllers/messenger.controller');

router.get(/\/([A-Za-z0-9]+)\/page(\d+)/, middlewareCheckGoogleProfile, middlewareCheckEmail, getMemberChat);

module.exports = router;

File: backend/routes/messenger.route.go
package routes

import (
	"github.com/gofiber/fiber/v2"
	controller "github.com/simopunkc/chirpbird-v2/controllers"
)

func MessengerRoute(app fiber.Router) {
	app.Get("/:room/page:pages", controller.GetMemberChat)
}
